src/data/text/item_descriptions.h

```diff
@@ -753,126 +753,126 @@ static const u8 sIapapaBerryDesc[] = _(
 static const u8 sRazzBerryDesc[] = _(
     "{POKEBLOCK} ingredient.\n"
```

Comment:
Should `{POKEBLOCK}` be changed to just `Pokéblock`? I know it's wider but the only other way of decapsing this is to edit the font images.